common/workunit/workunit.hpp

```diff
@@ -359,10 +359,11 @@ enum WUFileType
     FileTypeHintXml = 3,
     FileTypeXml = 4,
     FileTypeLog = 5,
-    FileTypeSize = 6
+    FileTypePostMortem = 6,
```

Comment:
why insert before last here ?  Is there any compatibility issue with this ?

Reply:
FileTypeSize is used to mark the number of items in the enumeration - so it is correct